packages/cloud-cognitive/src/components/CreateSidePanel/_create-side-panel.scss

```diff
@@ -0,0 +1,79 @@
+//
+// Copyright IBM Corp. 2021, 2021
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '@carbon/import-once/scss/import-once';
+@import '../../global/styles/carbon-settings';
+@import '../../global/styles/project-settings';
+
+@import 'carbon-components/scss/components/button/button';
+
+@mixin create-side-panel {
+  $block-class: #{$pkg-prefix}--create-side-panel;
+
+  .#{$block-class}.exp--side-panel__container {
```

Comment:
Might be best to replace `.exp....` with `.#{$pgk-prefix}....` since this prefix may change in the future.

Or even better, create a new sass variable that has the side panel block class.
```scss
$side-panel-block-class: #{$pkg-prefix}--side-panel__container
```